### PR TITLE
Set images versions of event-sources component

### PR DESCRIPTION
**Pull request by antoineco**
**Description**

Changes proposed in this pull request:

- Set images versions in Helm chart values ([corresponding Prow jobs](https://status.build.kyma-project.io/?type=postsubmit&job=post-kyma-components-event-sources))

**Related issue(s)**

issue #6288
pr #6296

### CODE REVIEW

File: resources/event-sources/values.yaml [hunk 1]
```diff
@@ -4,9 +4,9 @@ global:
   controllerManager:
     name: controller-manager
     image: event-sources-controller-manager
-    dir: develop/
-    version: #TODO
+    dir: ''
```
Comment (sayanh) [inline resources/event-sources/values.yaml]: I think we need to be consistent with the other images which have `develop/` in their path.

Comment (antoineco) [reply resources/event-sources/values.yaml]: The Prow job sends an empty dir and builds images like that. If I add that dir here, the image name becomes invalid. I think this changed with the shared Makefile.

Comment (sayanh) [reply resources/event-sources/values.yaml]: True

File: resources/event-sources/values.yaml [hunk 2]
```diff
@@ -4,9 +4,9 @@ global:
...
+    version: dc48c27c
   httpAdapter:
     image: event-sources-http-adapter
-    dir: develop/
-    version: #TODO
+    dir: ''
+    version: 0b31784e
```
Comment (k15r) [inline resources/event-sources/values.yaml]: there is also this version: [dc48c27c](https://console.cloud.google.com/gcr/images/kyma-project/EU/event-sources-http-adapter@sha256:44b3283569284d9b31584ad30da839a118ebcee7b07de7531aac4e20f9c40471/details?tab=info)` for the event-sources-http-adapter
I think - even if no changes to this particular code have been introduced - we should use this as it is the latest version

Comment (antoineco) [inline resources/event-sources/values.yaml]:
```suggestion
    version: dc48c27c
```

Comment (antoineco) [reply resources/event-sources/values.yaml]: Done

**Pull request merged**